src/api/v3/positions.ts

```diff
@@ -0,0 +1,487 @@
+        maxPrice: maxPrice, // max price (Y/X)
+        fees: {
+          x: new BigNumber(
+            fees.x.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenX.decimals))
+          ).toString(),
+          y: new BigNumber(
+            fees.y.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenY.decimals))
+          ).toString(),
+        },
+        feesDollar: new BigNumber(
+          fees.x.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenX.decimals))
+        )
+          .multipliedBy(tokenPrices[tokenXSymbol])
+          .plus(
+            BigNumber(
+              fees.y.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenY.decimals))
+            ).multipliedBy(tokenPrices[tokenYSymbol])
+          )
+          .toString(),
+        isInRange: isInRange,
+      }); */
+      return {
+        position: position,
+        currentTickIndex: contractStorageParameters.currTickIndex,
+        tokenX: tokenX,
+        tokenY: tokenY,
+        liquidity: {
+          x: new BigNumber(
+            liquidity.x.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenX.decimals))
+          ),
+          y: new BigNumber(
+            liquidity.y.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenY.decimals))
+          ),
+        },
+        liquidityDollar: new BigNumber(
+          liquidity.x.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenX.decimals))
+        )
+          .multipliedBy(tokenPrices[tokenX])
+          .plus(
+            BigNumber(
+              liquidity.y.dividedBy(
+                new BigNumber(10).pow(contractStorageParameters.tokenY.decimals)
+              )
+            ).multipliedBy(tokenPrices[tokenY])
+          ),
+        minPrice: minPrice, // min price (Y/X)
+        maxPrice: maxPrice, // max price (Y/X)
+        fees: {
+          x: new BigNumber(
+            fees.x.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenX.decimals))
+          ),
+          y: new BigNumber(
+            fees.y.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenY.decimals))
+          ),
+        },
+        feesDollar: new BigNumber(
+          fees.x.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenX.decimals))
+        )
+          .multipliedBy(tokenPrices[tokenX])
+          .plus(
+            BigNumber(
+              fees.y.dividedBy(new BigNumber(10).pow(contractStorageParameters.tokenY.decimals))
+            ).multipliedBy(tokenPrices[tokenY])
+          ),
+        isInRange: isInRange,
+        isMaxPriceInfinity:
+          parseInt(position.upper_tick_index) ==
+          Tick.nearestUsableTick(MAX_TICK, contractStorageParameters.tickSpacing),
+      };
+    });
+
+    return await Promise.all(finalPositionPromise);
+  } catch (error) {
+    console.log("v3 position error: ", error);
+  }
+};
+
+export const createIncreaseLiquidityOperation = async (
```

Comment:
Operation in API folder? @AniketSindhu

Reply:
This is not called from the frontend directly, The main operation is in the operation folder which internally calls this